packages/database/src/utilities/migration.js

```diff
@@ -18,14 +18,18 @@ export const insertObject = async (db, table, data, onError) =>
   db.insert(table, Object.keys(data), Object.values(data), onError);
 
 export const deleteObject = async (db, table, condition) => {
-  const [key, value] = Object.entries(condition)[0];
+  const where = Object.entries(condition)[0]
```

Comment:
It looks like you're running the map on the first `[key, value]` array rather than the parent array of `[key, value]` pairs

```suggestion
  const where = Object.entries(condition)
```